packages/connector-jsdom/src/connector.ts

```diff
@@ -31,20 +31,18 @@ import { fork, ChildProcess } from 'child_process';
 
 import { JSDOM, ResourceLoader, VirtualConsole } from 'jsdom';
 
-import { debug as d } from 'hint/dist/src/lib/utils/debug';
+import { debug as d, HttpHeaders, network } from '@hint/utils';
 import { getContentTypeData, getType } from 'hint/dist/src/lib/utils/content-type';
 import {
-    HttpHeaders,
     HTMLDocument,
     IConnector,
     HTMLElement,
     Event, FetchEnd, FetchError,
     NetworkData
 } from 'hint/dist/src/lib/types';
 import { Engine } from 'hint/dist/src/lib/engine';
-import isHTMLDocument from 'hint/dist/src/lib/utils/network/is-html-document';
-import createHTMLDocument from 'hint/dist/src/lib/utils/dom/create-html-document';
-import traverse from 'hint/dist/src/lib/utils/dom/traverse';
+import { createHTMLDocument } from 'hint/dist/src/lib/utils/dom/create-html-document';
```

Comment:
I thought we were going to avoid importing from paths like this one 🤔
Maybe the `dom` utils should be moved to `@hint/utils`?

Reply:
> Maybe the dom utils should be moved to @hint/utils?

We can't move this. There is some utils that can't be moved from `hint` due to circular references.

Reply:
We can't link to `dist/src/lib` so if it can't be moved then we need to expose this more easily. Otherwise it will be considered a private API 😟

Reply:
sorry, I was only answering about moving dom to `@hint/utils`. We are exposing the utils, and I'm working on using just 'hint' :)